eng/common/templates/jobs/build-images.yml

```diff
@@ -121,7 +126,8 @@ jobs:
     displayName: Publish Image Info File Artifact
   - ${{ if and(eq(variables['System.TeamProject'], parameters.internalProjectName), ne(variables['Build.Reason'], 'PullRequest')) }}:
       # Define the task here to load it into the agent so that we can invoke the tool manually
-    - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+      # TODO: Revert the build-specific pinned version when https://github.com/dotnet/docker-tools/issues/1152 is fixed
+    - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0.197.56
```

Comment:
This is the change that improves build reliability. I pulled it out in https://github.com/microsoft/go-images/pull/236 in case the rest of this upgrade caused any issues, but CI is green, so we can do the full upgrade. Keeping fully in sync is better.

The release made it through without this, but I'd like to get up to date for next time. This is just getting our `eng/common` infra in sync with what's been working for .NET Docker, so there's no need for in depth review.